pkgs/applications/misc/lutris/default.nix

```diff
@@ -40,7 +40,7 @@ in buildPythonApplication rec {
     sha256 = "1g093g0difnkjmnm91p20issdsxn9ri4c56zzddj5wfrbmhwdfag";
   };
 
-  nativeBuildInputs = [ wrapGAppsHook ];
+  nativeBuildInputs = [ wrapGAppsHook gtk3 glib ];
```

Comment:
put these in `buildInputs`, these are runtime dependencies.
See also https://nixos.org/nixpkgs/manual/#sec-language-gnome

Reply:
I tried it both ways and it seemed like they only added their environment variables when in `nativeBuildInputs`, but I don't understand why. I don't experience the issue though, so I can't test it properly. I've moved them and will ask aanderse to check both ways if it doesn't work.